specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/cluster.json

```diff
@@ -930,6 +975,10 @@
           "format": "int32",
           "description": "The instance count of the cluster."
         },
+        "VMGroupName": {
```

Comment:
I know this property name is not CamelCase, but in order to fix s360 issues, I need to change this to keep consistent with backend.
Our service will GA a new api version in April, in the new api version, I will fix this issue. Now I can create a work item or github issue to track this.
If it is ok, I hope this can check in. @JeffreyRichter You opinion is important for us, I will follow your suggestion. How do you like?

Reply:
We are approving breaking changes until the end of January 2021 so that the swager matches the service. So, if the service uses VMGroupName, then the service is wrong (it should have been vmGroupName) but it is too late now and the service must forever be wrong. And so the swagger must also be wrong. You should get an exception from the S360 rule.